openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -219,17 +222,19 @@ struct UhPartitionInner {
     #[inspect(with = "inspect::AtomicMut")]
     no_sidecar_hotplug: AtomicBool,
     use_mmio_hypercalls: bool,
+    backing_shared: BackingShared,
 }
 
-#[derive(Clone, Inspect)]
+#[derive(Inspect)]
 #[inspect(external_tag)]
-enum BackingShared {
+#[doc(hidden)]
```

Comment:
Why?

Reply:
Everything referenced in `BackingPrivate` has to be `pub`, but this isn't part of the public API of `virt_mshv_vtl`.

Maybe I should move this into a `mod private` to avoid this, I dunno.

I really wish you could just have private traits in Rust without playing these games.